modules/ROOT/pages/zuora/zuora-connector-examples.adoc

```diff
@@ -0,0 +1,68 @@
+= Zuora Examples - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+== Example: Retrieve Account Details for an Account
+
+== Studio Flows
+
+* Listener (HTTP) - Accepts data from HTTP requests.
+* Zuora API Connector - Connects with Zuora and executes a query to retrieve the account details for the specified account ID.
+* Transform message - Outputs the results of the `Get account` operation in JSON format.
+
++
+image::zuora/zuora-studio-use-case.png[studio_use_case]
```

Comment:
Would question how necessary this image is here. It does not seem to add anything beyond the text description.